packages/lodestar-cli/src/util/file.ts

```diff
@@ -21,7 +21,7 @@ export function parse<T=Json>(contents: string, fileFormat: FileFormat): T {
     case FileFormat.json:
       return JSON.parse(contents);
     default:
-      throw new Error("Invalid filetype");
+      throw Error("Invalid filetype");
```

Comment:
`new` got omitted off here, this should still be `throw new Error`, same with the error in `stringify` in this file.

Reply:
fixed